Libraries/LibWeb/DOM/Node.h

```diff
@@ -110,7 +110,7 @@ class Node : public EventTarget {
 
     NodeType type() const { return m_type; }
     bool is_element() const { return type() == NodeType::ELEMENT_NODE; }
-    bool is_text() const { return type() == NodeType::TEXT_NODE; }
+    bool is_text() const { return type() == NodeType::TEXT_NODE || type() == NodeType::CDATA_SECTION_NODE; }
```

Comment:
Maybe use the helper, which is also used in `is_character_data()` below, so:  `first_is_one_of(type(), NodeType::TEXT_NODE, NodeType::CDATA_SECTION_NODE);`